### ISSUE TITLE
Show/hide Form component

### ISSUE THREAD

**Issue by gacsphere** [gacsphere/capstone-project#19]
## Value Proposition
As a user, 
I want to be able to create new nest boxes
without navigating away from the homepage.
## Description

![userstory11_Form-toggle](https://user-images.githubusercontent.com/109087424/199452114-d248532c-b4ff-40d0-ba86-4ab95af1683f.png)
![userstory12_Form-toggle](https://user-images.githubusercontent.com/109087424/199452119-fb78b62c-24be-4f9c-b9d8-c60dbee7be36.png)

## Acception Criteria
- [ ] Display Form when needed.
- [ ] Hide Form when not needed or when done with entering data.

## Tasks
- [ ] Write function for showing and hiding Form component.
- [ ] Add a button for showing the Form.
- [ ] Implement function on Form and button.

**Comment by Kheirah** [comment]
Looks good!

I'd rephrase the value proposition a bit:
As a user,
I want to be able to create new nest boxes
without navigating away from the homepage.

I'm eagerly awaiting the new feature 🥳
